### PR TITLE
AR-619: Backend layout

### PR DESCRIPTION
**Pull request by yepzdk**
https://jira.itkdev.dk/browse/AR-619

### CODE REVIEW

File: src/components/navigation/nav-items/nav-items.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import { React } from "react";
+import { Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  // faTachometerAlt,
```
Comment (sinejespersen) [inline src/components/navigation/nav-items/nav-items.jsx:7]: Why this?

File: src/components/navigation/topbar/topbar.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  faDesktop,
+  faStream,
+  faQuestionCircle,
+  faUserCircle,
+  faSignOutAlt,
+} from "@fortawesome/free-solid-svg-icons";
+import NavItems from "../nav-items/nav-items";
+import "./topbar.scss";
+/**
+ * The side bar component.
```
Comment (sinejespersen) [inline src/components/navigation/topbar/topbar.jsx:21]: update comments

**Review by sinejespersen — APPROVED** [review]

File: src/components/navigation/nav-items/nav-items.jsx (hunk 1 above)
Comment (yepzdk) [reply src/components/navigation/nav-items/nav-items.jsx:7]: It's related to the dashboard. I have updated the comment.

File: src/components/navigation/topbar/topbar.jsx (hunk 2 above)
Comment (yepzdk) [reply src/components/navigation/topbar/topbar.jsx]: Comment updated.

**Pull request merged**